src/pl/plpgsql/src/pl_exec.c

```diff
@@ -260,6 +260,10 @@ plpgsql_exec_function(PLpgSQL_function *func, FunctionCallInfo fcinfo,
 	ErrorContextCallback plerrcontext;
 	int			i;
 	int			rc;
+	bool orig_gp_enable_gpperfmon = gp_enable_gpperfmon;
+
+	if (log_min_messages > DEBUG5)
```

Comment:
Why DEBUG5 in opposite to other places with DEBUG4?

Reply:
Fixed